src/main/scala/org/broadinstitute/dsde/firecloud/core/ExportEntitiesByType.scala

```diff
@@ -93,14 +93,14 @@ class ExportEntitiesByTypeActor(requestContext: RequestContext) extends Actor wi
       Future(RequestComplete(StatusCodes.BadRequest)) pipeTo context.parent
   }
 
-  private def getZipBytes(filename: String, membershipData: String, rootType: String, rootData: String): Array[Byte] = {
+  private def getZipBytes(entityType: String, membershipData: String, entityData: String): Array[Byte] = {
     val bos = new ByteArrayOutputStream()
     val zos = new ZipOutputStream(bos)
-    zos.putNextEntry(new ZipEntry(filename))
+    zos.putNextEntry(new ZipEntry(entityType + "_membership.txt"))
     zos.write(membershipData.getBytes)
     zos.closeEntry()
-    zos.putNextEntry(new ZipEntry(rootType + ".txt"))
-    zos.write(rootData.getBytes)
+    zos.putNextEntry(new ZipEntry(entityType + "_entity.txt"))
```

Comment:
I'm not sure if the _entity is necessary on the file name, but it is confusing to need two files per set, so maybe it's better to add that to the end instead of just calling it "sample_set.txt"